src/main/java/com/spotinst/sdkjava/model/requests/admin/account/messageCenter/ListAllMessagesRequest.java

```diff
@@ -0,0 +1,88 @@
+package com.spotinst.sdkjava.model.requests.admin.account.messageCenter;
```

Comment:
Incorrect package. Should be in requests.admin.messageCenter.

Reply:
Resolved